include/powerloader/powerloader.hpp

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) 2021, QuantStack and Mamba Contributors
```

Comment:
Should be `2022`

Reply:
Nope, since we already have commits on the repo in 2021. I will make a pass to add the copyright to all files when I'm sure it won't conflict.